extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -879,7 +878,10 @@ export class AmpA4A extends AMP.BaseElement {
         creative.slice(metadataStart + METADATA_STRING.length, metadataEnd));
       const ampRuntimeUtf16CharOffsets =
         metaDataObj['ampRuntimeUtf16CharOffsets'];
-      if (!isValidOffsetArray(ampRuntimeUtf16CharOffsets)) {
+      if (!isArray(ampRuntimeUtf16CharOffsets) ||
+          ampRuntimeUtf16CharOffsets.length != 2 &&
```

Comment:
Why did you inline this and get rid of the function?  Just b/c it's only used here these days?  I still think it makes the code more readable, but if you'd rather kill it, no problem.  But I think you want to convert all of the `&&` to `||` here.